cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControl.java

```diff
@@ -42,6 +42,10 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static com.linkedin.kafka.cruisecontrol.KafkaCruiseControlState.SubState.EXECUTOR;
+import static com.linkedin.kafka.cruisecontrol.KafkaCruiseControlState.SubState.ANALYZER;
+import static com.linkedin.kafka.cruisecontrol.KafkaCruiseControlState.SubState.MONITOR;
```

Comment:
maybe we can change to more concise statement
`import static com.linkedin.kafka.cruisecontrol.KafkaCruiseControlState.SubState.*`